### ISSUE TITLE
"Who is using svelte" section ?

### ISSUE THREAD

**Issue by infuzz** [svelte-society/sveltesociety.dev#18]
Since "who is using svelte" svelte.dev section has been removed to keep only major users, it could make sense to have this kind of section put in svelte society website ? 
It was very interesting to see how svelte was spreading the world. 
Now we can't have this information. 
Yes, see that IBM,  Go daddy and 13 others users is pretty cool, but see that more than 300+ took the train already was also a good indicator how svelte is really good.

Maintain and check each pr about that is probably time consuming.
Maybe there is another way to keep track of this without using git repo.

**Comment by benmccann** [comment]
I think some other ways of demonstrating this that are easier to digest would be showing that [Svelte is downloaded about ~8m times a year](https://www.npmjs.com/package/svelte) and showing that it's the framework that people are [most satisfied with and most interested in learning about](https://2020.stateofjs.com/en-US/technologies/front-end-frameworks/#front_end_frameworks_experience_ranking). This would be a much larger amount of data and in my mind would give more confidence than showing some logos no one is familiar with

**Comment by infuzz** [comment]
Yes, you are right, figures and links you provided are definitely a very good and better indicator to show the svelte adoption ratio. What was interesting in links back to logos & websites of those who adopted svelte was diversity of companies size, early adoption and finally how they contribute to spread svelte. Maybe also a kind of SEO good practice for sveltesociety site and for linked references.

**Comment by kevmodrome** [comment]
We could also show the recent StackOverflow survey results somewhere 🤔

**Comment by shahul01** [comment]
we could also use a line chart with time vs no of download to show the increasing svelte trend

**Comment by brittneypostma** [comment]
I love this idea and would make a great addition to the home page imo.

**Comment by benmccann** [comment]
Yeah, it'd be really nice to showcase a chart like that and the StackOverflow / State of JS survey results. I think they could go on the main svelte.dev site actually

**Comment by benmccann** [comment]
We're adding some of this to the svelte.dev homepage as tracked in https://github.com/sveltejs/svelte/issues/6791

**Comment by brittneypostma** [comment]
Do you think a similar line like what got merged to svelte.dev should be added to our intro page?

**Comment by benmccann** [comment]
No objection to it, but personally I don't really think it's necessary and feels a little off brand
